src/gluonts/model/seq2seq/_mq_dnn_estimator.py

```diff
@@ -237,11 +240,12 @@ def __init__(
             prefix="decoder_",
         )
 
-        quantile_output = (
-            QuantileOutput(self.quantiles, is_iqf=self.is_iqf)
-            if self.quantiles
-            else None
-        )
+        if not self.quantiles:
+            quantile_output = None
+        elif is_iqf:
+            quantile_output = IncrementalQuantileOutput(self.quantiles)
```

Comment:
Thanks this is nice and cleaner to have a separate class.

Reply:
Yes I think this is better than a boolean flag. By the way, we could use this also in TFT to get rid of quantile crossing there too.

Reply:
Yes, this should be used a generic class that can be combined on the last layer of any model, such as TFT, to resolve quantile crossing, not just specific to MQ-CNN.